docs/How-We-Work/Secrey-Key-Mgmt.md

```diff
@@ -0,0 +1,43 @@
+# Secret Key Management Strategy
+
+Moving forward, the dev team will be taking additional precautions to protect the TDP system from unauthorized access. 
+
+The goal of these efforts is to help prevent unintended secret key leakage. Though it isn't possible to completely eliminate the possibility of secret key leakage, the strategies documented herein will help improve incident response in the event where keys are leaked. 
+
+## What are secret keys, and how can these be leaked?
+
+As described [here](https://github.com/raft-tech/TANF-app/blob/raft-tdp-main/docs/Architecture%20Decision%20Record/004-configuration-by-environment-variable.md), the TDP system is configured by environment variables. [Environment variables](https://medium.com/chingu/an-introduction-to-environment-variables-and-how-to-use-them-f602f66d15fa) help minimize the risk of running into unexpected issues when changes are made to the system. Some of these environment variables are senstive because these are used to authorize access to TDP. We refer to these as "Secret keys". 
+
+If secret keys are leaked/exposed, this can not only change how the TDP system functions, but this also compromises the security of the data files flowing through the system (and the individuals and families represented in these data). There are two primary avenues that secret key leakage can happen: 
+- a security breach in one or more of the project [tools](https://about.codecov.io/security-update/) that use the secret keys(*relatively speaking, this is more challenging to prevent*), or 
```

Comment:
```suggestion
- a security breach in one or more of the project tools (e.g. GitHub, CircleCi, [CodeCov](https://about.codecov.io/security-update/), Cloud.gov, Docker) that use or rely on secret keys  (*relatively speaking, this is challenging to prevent*), or 
```